ui/apps/platform/cypress/integration/dashboard.test.js

```diff
@@ -53,7 +53,7 @@ describe('Dashboard page', () => {
         cy.get('@riskTiles').last().click();
         cy.location().should((location) => {
             expect(location.pathname).to.eq(violationsUrl);
-            expect(location.search).to.eq('?severity=LOW_SEVERITY');
+            expect(location.search).to.eq('?search[Severity]=LOW_SEVERITY');
```

Comment:
Would be great to harden these types of tests for the new dashboard, by getting the unfiltered count on the destination pages, and then making sure the count is less when we visit the filtered URLs.